transaction/core/src/membership_proofs/errors.rs

```diff
@@ -1,29 +1,31 @@
 // Copyright (c) 2018-2020 MobileCoin Inc.
 
 use crate::range;
-use failure::Fail;
+use displaydoc::Display;
 
 /// Reasons why a creating or validating a proof of membership might fail.
-#[derive(Debug, Fail, PartialEq)]
+#[derive(Debug, Display, PartialEq)]
```

Comment:
changed this to displaydoc, because as a policy we are trying to migrate to displaydoc, and so anyhting using failure that we touch should change to displaydoc